.eslintrc.json

```diff
@@ -1,7 +1,17 @@
 {
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:react/recommended",
+    "plugin:jsx-a11y/recommended",
+    "plugin:react-hooks/recommended",
+    "plugin:jest/recommended",
+    "plugin:testing-library/react",
+    "plugin:react/jsx-runtime"
+  ],
   "root": true,
   "ignorePatterns": ["**/*"],
-  "plugins": ["@nx"],
+  "plugins": ["@nx", "@typescript-eslint", "import", "sort-keys-fix"],
```

Comment:
I did not bring over the original `.eslintrc.json` settings for `ignorePatterns`, `parser`, `parserOptions`, and `env`, let me know if they are required but they did not affect my initial formatting test
Original Config: https://github.com/projectNEWM/newm-web/blob/19dc616045fbf9f399a50beaabe2ca5689689ae1/.eslintrc.js